workbench-core/accounts/src/models/projects/createProjectRequest.ts

```diff
@@ -2,14 +2,14 @@
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
-import { z } from 'zod';
+import { resourceTypeToKey, z } from '@aws/workbench-core-base';
 
 // eslint-disable-next-line @rushstack/typedef-var
 export const CreateProjectRequestParser = z
   .object({
-    name: z.string(),
-    description: z.string(),
-    costCenterId: z.string()
+    name: z.string().swbName().required(),
+    description: z.string().swbDescription().required(),
+    costCenterId: z.string().swbId(resourceTypeToKey.costCenter.toLowerCase()).required()
```

Comment:
Create a new ZodString interface function for `ccId` in `workbench-core/base/src/utilities/validatorHelper.ts`